testproject/Assets/Tests/Manual/Scripts/GenericNetworkObjectBehaviour.cs

```diff
@@ -86,8 +86,7 @@ private void OnTriggerEnter(Collider other)
                 }
                 else
                 {
-                    NetworkObject.Despawn();
-                    NetworkObject.gameObject.SetActive(false);
```

Comment:
did we drop this intentionally?

Reply:
Review it again as I just updated.
It should make more sense...